### DISCUSSION TITLE
Issue with link elements

### DISCUSSION THREAD

**Discussion by BTrifonov** [clientIO/joint#2343] [Q&A]
### Introduction

Hi, 
when creating a joint.shapes.standard.Link and attaching a callback function on the 'link: pointerclick' or 'link:pointerdblclick' event I am getting the following error, when the event is triggered:
     "Uncaught Error: Illegal access. Moveto segments should not need a start property Error"
On the link I have attached the following tools:
-  joint.linkTools.Vertices
-  joint.linkTools.TargetArrowhead
- joint.linkTools.TargetAnchor

- joint.linkTools.SourceAnchor
- joint.linkTools.SourceArrowhead

- joint.linkTools.Boundary
- joint.linkTools.Remove
As this project is part of my bachelor thesis, any help is appreciated!
Kind regards, 
Boris 

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
Please create a JSFiddle that demonstrates the issue. Thanks.

**Comment by sidnaik03** [comment]
I am encountering the same error. @BTrifonov, have you found any solution to the issue? The console indicates that the problem originates from this line: [here](https://github.com/clientIO/joint/blob/6d12e652bbf133b5e6f85b3b4889218258604610/packages/joint-core/src/g/path.mjs#L2119)

**Reply by sidnaik03** [reply]
I was using Vue 3. When I applied the Vue WATCH to the graph object, it resulted in an error; however, removing it made the error disappear. It's peculiar how it's behaving in my case.
